util/src/main/java/com/google/cloud/hadoop/util/interceptors/InvocationIdInterceptor.java

```diff
@@ -70,7 +73,7 @@ public void intercept(HttpRequest request) throws IOException {
       } else {
         newValue = invocationEntry;
       }
-      GoogleCloudStorageEventBus.onGcsRequest(new GcsRequestExecutionEvent());
```

Comment:
why we have removed this?

Reply:
tracker.init(request); in line 54 take care of this.

Reply:
yes, this is getting used from gRPC.